Some third-party packages were deleted from examples/imports/
so tests that depend on those third-party packages are excluded
--- a/ts/readme.test.ts
+++ b/ts/readme.test.ts
@@ -195,6 +195,7 @@
       assert.strictEqual(consoleLogCalls[0].join(" "), "Result: EXAMPLE FILE CONTENT")
     })
 
+    /**
     it("asyncify module loader example", async () => {
       {
         const module = await newQuickJSAsyncWASMModule()
@@ -232,6 +233,7 @@
       const lastLogMessage = consoleLogCalls[consoleLogCalls.length - 1].join(" ")
       assert.strictEqual(lastLogMessage, "<div><strong>Hello world!</strong></div>")
     })
+    **/
 
     it("asyncify file reading example", async () => {
       {
